index.html

```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
             <img src="resources/image_geometry_2.svg" alt="Geometrical decoration 2">
```

Comment:
Kindly, decoration images should be part of the background of your site, I recommend using background-image

Reply:
Kindly, by convention the FOOTER semantic tag is used in the last part of your website, in your case I recommend using DIV